lighthouse-core/audits/time-to-interactive.js

```diff
@@ -79,6 +79,10 @@ class TTIMetric extends Audit {
 
       const fmpTiming = fMPtsInMS - navStartTsInMS;
 
+      const traceEndTsInMS = trace.traceEvents.reduce(
+        (acc, traceEvent) => Math.max(acc, traceEvent.ts), 0) / 1000;
```

Comment:
lgtm. we have tabTrace.processEvents, which is already sorted, but it's missing other processes (like browser, in particular). After considering it, we definitely want the last last TS in the trace, regardless of process. So let's do it this way. :)

Also this Math.max reduce is crazy fast. On a 61MB trace, this completed in 15ms. :)  Meanwhile a sort() took ~160ms.